bridge/docs/block_relay.md

```diff
@@ -0,0 +1,126 @@
+# Block Relay in Witnet
```

Comment:
In this section,it may be interesting to reinforce the following points:
- The block relay contract is alike to a Solidity implementation of a Witnet lightweight node, inasmuch it only contains the block headers and performs little to no validations.
- As a lightweight node, the block relay inherits the challenges and trade-offs thereof, specially in terms of trust mitigation and probabilistic finality.
- Proof of Inclusion is equivalent to SPV.